pkg/controllers/virtualmachineinstance_controller.go

```diff
@@ -54,7 +54,9 @@ func (r *VirtualMachineInstanceReconciler) Reconcile(ctx context.Context, reques
 		// Error reading the object - requeue the request.
 		return ctrl.Result{}, err
 	}
-	err = r.ZoneManager.UpdateZone(request.NamespacedName, FilterMultusNonDefaultInterfaces(vmi.Status.Interfaces, vmi.Spec.Networks))
+	filteredInterfaces := FilterMultusNonDefaultInterfaces(vmi.Status.Interfaces, vmi.Spec.Networks)
+	filteredInterfaces = FilterInterfacesWithNoName(filteredInterfaces) // The interface/network name is used to build the FQDN, therefore, interfaces reported without a name are filtered out
```

Comment:
Lets remove please the comment, it is pretty trivial here
also comment should be line above the code usually imo

Reply:
I find the comment useful and not trivial.
Moved it to be above the code.

Reply:
Imo it is very trivial and the comment is redundant
code should be (and it is) self explanatory